content/en/docs/ops/configuration/mesh/app-health-check/index.md

```diff
@@ -75,10 +76,10 @@ liveness-6857c8775f-zdv9r        2/2       Running   0           4m
 
 ## Liveness and readiness probes using the HTTP request approach
```

Comment:
HTTP request/TCP probe?

Reply:
We changed it to "Liveness and readiness probes using the HTTP or TCP approach" not to make the heading even longer.

Hope thats fine for you.

Reply:
👍